File: python/Athirson.py
class Me:
    def __init__(self, name, role, workplace, languages):
        self.__name = name
        self.__role = role
        self.__workplace = workplace
        self.__languages = languages

    def setName(self, name) -> None:
        self.__name = name

    def getName(self) -> str:
        return self.__name

    def setRole(self, role) -> None:
        self.__role = role

    def getRole(self) -> str:
        return self.__role

    def setWorkplace(self, workplace) -> None:
        self.__workplace = workplace

    def getWorkplace(self) -> str:
        return self.__workplace

    def setLanguages(self, languages) -> None:
        self.__languages = languages

    def getLanguages(self) -> str:
        return self.__languages


me = Me('Athirson Silva', 'IT Support intern',
        'Colégio Rio Branco', ['pt_BR', 'en_US'])

# Output
print(me.getName())
print(me.getRole())
print(me.getWorkplace())
print(me.getLanguages())

# Input
me.setName('Athirson Silva')
me.setRole('IT Support intern')
me.setWorkplace('Colégio Rio Branco')
me.setLanguages(['pt_BR', 'en_US'])

File: typescript/Athirson.ts
interface IEmployee {
	getWorkplace(): string;
	getAmbition(): string;
	getRole(): string;
	setWorkplace(workplace: string): void;
	setAmbition(ambition: string): void;
	setRole(role: string): void;
}

class Me {
	private workplace: string;
	private role: string;
	private ambition: string;
	private languages: string[];

	constructor(
		workplace: string,
		role: string,
		ambition: string,
		languages: string[]
	) {
		this.workplace = workplace;
		this.role = role;
		this.ambition = ambition;
		this.languages = languages;
	}

	public getWorkplace = (): string => {
		return this.workplace;
	};

	public getAmbition = (): string => {
		return this.ambition;
	};

	public getRole = (): string => {
		return this.role;
	};

	public getLanguages = (): string[] => {
		return this.languages;
	};

	public setWorkplace = (workplace: string): void => {
		this.workplace = workplace;
	};

	public setAmbition = (ambition: string): void => {
		this.ambition = ambition;
	};

	public setRole = (role: string): void => {
		this.role = role;
	};

	public setLanguages = (languages: string[]): void => {
		this.languages = languages;
	};
}

const me = new Me(
	'Colégio Rio Branco',
	'IT Support intern',
	'Become a backend developer',
	['pt_BR', 'en_US']
);

// Output
me.getWorkplace();
me.getAmbition();
me.getRole();
me.getLanguages();

// Input
me.setWorkplace('Colégio Rio Branco');
me.setAmbition('Contribute to open source projects');
me.setRole('IT Support intern');
me.setLanguages(['pt_BR', 'en_US']);
